docs/examples/policies/chainloop-commit.yaml

```diff
@@ -21,45 +21,44 @@ spec:
     - kind: ATTESTATION
       embedded: |
         package main
-        
+
         import rego.v1
-                  
+
         ################################
         # Common section do NOT change #
         ################################
-        
+
         result := {
-          "skipped": skipped,
-          "violations": violations,
-          "skip_reason": skip_reason,
+        	"skipped": skipped,
```

Comment:
why do we have more spaces now? 

Do ew want to have 4 instead of 2? I personally like 2 spaces, any reason for using 4?